controllers/openshift_controller.go

```diff
@@ -783,7 +783,7 @@ func (r *KataConfigOpenShiftReconciler) createRuntimeClass(runtimeClassName stri
 			ObjectMeta: metav1.ObjectMeta{
 				Name: runtimeClassName,
 			},
-			Handler: runtimeClassName,
+			Handler: handler,
```

Comment:
Maybe worth clarifying in the commit message the need to do this, e.g. "the upcoming kata-cc runtime class will be using different TEE-specific handlers"